osquery/tables/events/windows/windows_events.cpp

```diff
@@ -61,14 +61,24 @@ REGISTER(WindowsEventSubscriber, "event_subscriber", "windows_events");
 /// Helper function to recursively parse a boost ptree
 void parseTree(const pt::ptree& tree, std::map<std::string, std::string>& res) {
   for (const auto& node : tree) {
-    auto nodeName = node.second.get("<xmlattr>.Name", "");
+    // Skip this since it's not actually part of the EventData. Also prevents
+    // us from adding every Name attribute into its own key invalidly. This is
+    // part of a quirk of boost::ptree and its parsing of XML.
+    if (node.first == "<xmlattr>") {
+      continue;
+    }
 
+    auto nodeName = node.second.get("<xmlattr>.Name", "");
     if (nodeName.empty()) {
       nodeName = node.first.empty() ? "DataElement" : node.first;
     }
-    res[nodeName] = res[nodeName] == ""
-                        ? node.second.data()
-                        : res[nodeName] + "," + node.second.data();
+
```

Comment:
Why did you change this from a ternary?

Reply:
In the initial change I made, I thought I was going to use something within the if statements. I left it as is after cleaning up the solution.

Happy to change it back if it's a stylistic thing.

Site note: I find ternary expressions hard to read, which is why I expanded it at all.